pkg/sdk/identifier_parsers.go

```diff
@@ -239,10 +269,11 @@ func ParseFunctionArgumentsFromString(arguments string) ([]DataType, error) {
 			}
 			dataTypes = append(dataTypes, DataType(vectorDataType))
 		default:
-			dataType, err := stringBuffer.ReadString(',')
+			argument, err := stringBuffer.ReadString(',')
 			if err == nil {
-				dataType = dataType[:len(dataType)-1]
+				argument = argument[:len(argument)-1]
 			}
+			dataType := argument[strings.IndexRune(argument, ' ')+1:]
```

Comment:
is it the same as `strings.TrimSpace`?

Reply:
Add comment that it's for skipping arg name

Reply:
Done - moved above to have this behavior for vector types as well.